server/portal/apps/onboarding/api/views.py

```diff
@@ -170,6 +173,32 @@ def reset(self, request, setup_step):
         )
         setup_step.prepare()
 
+    def force_create_storage(self, request, setup_step):
+        """
+        Forces storage creation
+        """
+        if not request.user.is_staff:
+            raise PermissionDenied
+        setup_step.log("force_create_storage by {staff}".format(
+            step=setup_step.display_name(),
+            staff=request.user.username
+        )
+        )
+
+        # Mark the user's setup_complete as False
+        setup_step.user.profile.setup_complete = True
```

Comment:
Should this be `False`?

Reply:
@rstijerina I see. I would have to change this value to `True` after parsing the webhook, right?.

Reply:
Yeah, that would be ideal I think

Reply:
Added the handle [here](https://github.com/TACC/Core-Portal/blob/8239952b611d65df53a9f0e19105ef994a6669ab/server/portal/apps/system_creation/utils.py#L155)